bundle/Resources/translations/ezrepoforms_policies.en.xlf

```diff
@@ -0,0 +1,86 @@
+        <note>key: policy.limitation.identifier.newsection</note>
+      </trans-unit>
+      <trans-unit id="a9958f1d76f4685bf5fc32eee7514046db942160" resname="policy.limitation.identifier.newstate">
+        <source>New State</source>
+        <target>New State</target>
+        <note>key: policy.limitation.identifier.newstate</note>
+      </trans-unit>
+      <trans-unit id="be2b0449251f53db814dc4fbbb27e534419add2b" resname="policy.limitation.identifier.node">
+        <source>Location</source>
+        <target>Location</target>
+        <note>key: policy.limitation.identifier.node</note>
+      </trans-unit>
+      <trans-unit id="9daf8e1a94a6409f9b9b102e042c2858e4307ea8" resname="policy.limitation.identifier.owner">
+        <source>Owner</source>
+        <target>Owner</target>
+        <note>key: policy.limitation.identifier.owner</note>
+      </trans-unit>
+      <trans-unit id="e3773b71c8e7fda97f221ab25a6c25164d179d84" resname="policy.limitation.identifier.parentclass">
+        <source>Parent Content Type</source>
+        <target>Parent Content Type</target>
+        <note>key: policy.limitation.identifier.parentclass</note>
+      </trans-unit>
+      <trans-unit id="39bf0d9c1e0e8b64193bf09d83fbd89c2dac3421" resname="policy.limitation.identifier.parentdepth">
+        <source>Parent Depth</source>
+        <target>Parent Depth</target>
+        <note>key: policy.limitation.identifier.parentdepth</note>
+      </trans-unit>
+      <trans-unit id="cb52ff8582459d1bc981e9f98cf1c79876f83a1e" resname="policy.limitation.identifier.parentgroup">
+        <source>Parent Group</source>
+        <target>Parent Group</target>
+        <note>key: policy.limitation.identifier.parentgroup</note>
+      </trans-unit>
+      <trans-unit id="6208b97847450b255559e4a56c51ff06b76a08dd" resname="policy.limitation.identifier.parentowner">
+        <source>Parent Owner</source>
+        <target>Parent Owner</target>
+        <note>key: policy.limitation.identifier.parentowner</note>
+      </trans-unit>
+      <trans-unit id="bb77b27363dad566197a200290f0f0b18baa4705" resname="policy.limitation.identifier.section">
+        <source>Section</source>
+        <target>Section</target>
+        <note>key: policy.limitation.identifier.section</note>
+      </trans-unit>
+      <trans-unit id="548bc922c693c8259cbfd3354451a3035e2fa6f6" resname="policy.limitation.identifier.siteaccess">
+        <source>Site Access</source>
+        <target>Site Access</target>
+        <note>key: policy.limitation.identifier.siteaccess</note>
+      </trans-unit>
+      <trans-unit id="30cc18895dfb23b14d3f9b8249e90baf678c8180" resname="policy.limitation.identifier.state">
+        <source>State</source>
+        <target>State</target>
+        <note>key: policy.limitation.identifier.state</note>
+      </trans-unit>
+      <trans-unit id="2291e238752ad3c4d20ef08253f5d5736f6382d2" resname="policy.limitation.identifier.subtree">
+        <source>Subtree</source>
```

Comment:
Long version, if we add that, can be "Subtree of Location" or something.

Reply:
I agree `Subtree of Location` is better